README.md

```
@@ -71,146 +51,99 @@ docker-compose run evo-learn-test
+```python
+result = run_automl(
+    data_path="data.csv",
+    target_column="target",
+    config_path="config.yaml"
+)
 ```
 
-### Evaluation Outputs
+## Project Structure
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)